tool/github.sh

```diff
@@ -29,13 +29,13 @@ java -version
 
 # Get packages for the top-level grind script utilities.
 echo "pub get `pwd`"
-pub get
+dart pub get
 
 # Get packages for the test data.
-(cd flutter-idea/testData/sample_tests; echo "pub get `pwd`"; pub get)
+(cd flutter-idea/testData/sample_tests; echo "pub get `pwd`"; dart pub get)
```

Comment:
This file has been fixed in a PR awaiting review. If you could add 'dart ' to the two echo commands it would make merging conflict-free after I get the review. Not mandatory, since I can resolve the conflict myself.

Reply:
done!